www/classes/Model/User.php

```diff
@@ -155,6 +155,39 @@ public function update(string $name, int $perm_level, bool $active, int $alt_css
         return User::getById($this->id);
     }
 
+    public function combineUsers(User $user1): ?User
+    {
+        if ($user1->perm_level == 3) {
+            ErrorHandler::instance()->addError(sprintf('Kan beheerder \'%s\' niet mergen.', $user1->username));
+            return null;
+        }
+        Database::instance()->storeQuery("UPDATE article_reactions SET user_id = ? WHERE user_id = ?");
+        $stmt = Database::instance()->prepareStoredQuery();
+        $stmt->bind_param('ii', $this->id, $user1->id);
+        $stmt->execute();
+        if ($stmt->errno != 0) {
+            ErrorHandler::instance()->addError(sprintf('Kan \'%s\' niet mergen, conflict in reacties', $user1->username));
+            return null;
+        }
```

Comment:
Ik denk dat het ok is als de reacties van de oorspronkelijke gebruiker komen te vervallen als de doelgebruiker er al eentje heeft. Is niet perfect, maar anders wordt het denk ik snel onmogelijk.

Reply:
Zou je willen testen of je een query kunt maken die de reacties samenvoegt waar mogelijk, maar de emojis van de te verwijderen gebruikers verwijdert waar nodig? Ik denk dat je dat misschien kunt doen met `UPDATE IGNORE article_reactions SET user_id = ? WHERE user_id = ?` and dan nog een `DELETE` statement om de rest op te ruimen.